button.lua

```diff
@@ -47,11 +59,19 @@ function PrimeUI.button(win, x, y, text, action, fgColor, bgColor, clickedColor)
                     else action() end
                 end
                 -- Redraw the original button state.
+                win.setCursorPos(x, y)
+                win.setBackgroundColor(bgColor)
+                win.setTextColor(fgColor)
+                win.write(" " .. text .. " ")
+            elseif event == "timer" and timer == button and buttonDown then
```

Comment:
This should probably be moved with the `mouse_up` handler to reduce code duplication.

Reply:
I don't actually know why i did that :kekw: